.mvn/maven.config

```diff
@@ -1,4 +1,3 @@
---show-version
```

Comment:
I want to make `just` builds less verbose. I'm considering adding --show-version back into GitHub workflows.